jsk_pr2_robot/jsk_pr2_startup/jsk_pr2_warning/pr2_reset_motors.py

```diff
@@ -35,6 +35,8 @@ def motors_cb(self, msg):
         halted = msg.data
         rospy.logdebug("runstop: %s, halted: %s" % (self.run_stop, halted))
         if self.run_stop and halted:
+            rospy.logwarn("motor halted, but run stop is true")
+            rospy.logwarn("resetting motors")
```

Comment:
Is it best writing `resetting motors` here?
I suppose write `rospy.logwarn("resetting motors")` above line of `            self.reset_srv()`.

Reply:
Thanks, I updated.